src/python/grpcio/grpc/__init__.py

```diff
@@ -1447,6 +1447,20 @@ def stop(self, grace):
         """
         raise NotImplementedError()
 
+    def wait_for_termination(self, grace=None):
+        """Block current thread until the server stops.
+
+        The wait will not consume computational resources during blocking, and it
+        will block indefinitely. There are two ways to unblock:
+
+        1) Calling `stop` on the server in another thread;
+        2) The `__del__` of the server object is invoked.
+
+        Args:
+          grace: A duration of time in seconds or None.
```

Comment:
Can we define more clearly what `grace` does?

Reply:
The `grace` variable is removed. Now, the `wait_for_termination` function accepts no argument.

My original design is to replace our internal API which have both `delay` and `grace`. However, the semantic will become a lot messy. And it is hard to explain that:
1) The `grace` variable set here is not necessary the source of truth, because other thread can call `server.Stop` as well;
2) The `delay` variable will only work in main thread, what should it do in other thread;
3) The semantic of `delay` is hard to define that which signal should it react to.